### PR TITLE
Faq part two 

### PR DESCRIPTION
**Pull request by etpalmer63**
## Summary
This is the second part to #2340. This PR contains the unanswered questions. Help answering them is appreciated. 

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [x] include documentation in the code and/or rst files, if appropriate

### CODE REVIEW

File: Docs/sphinx_documentation/source/Faq.rst [hunk 1]
```diff
@@ -125,13 +125,50 @@ Source Documentation.) and
...
+**Q.** How does coarse-grained OpenMP parallelism work in AMReX? How is it
+different from the fine-grained approach? Moreover, what is the best
+approach to achieve maximum compute performance for hybrid
+MPI+OpenMP code?
+
+**A.** Our OpenMP strategy is explained in this paper, https://arxiv.org/abs/1604.03570.
+As for the ``AMREX_CRSEGRNDOMP`` macro, it was added to avoid nested OpenMP parallel regions
+when we started using the OpenMP threading over tiles approach. Currently, it's only used
+in a few places in the ``AmrLevel`` and ``StateData`` classes, where user provided functions are
+called.
```
Comment (WeiqunZhang) [inline Docs/sphinx_documentation/source/Faq.rst]: This macro has been removed.

Comment (etpalmer63) [reply Docs/sphinx_documentation/source/Faq.rst]: I think this question is a good opportunity to explain what AMReX offers. To address this I wrote the following. Let me know what you think.

Our OpenMP strategy is explained in this paper, https://arxiv.org/abs/1604.03570. Generally speaking, we work to ensure AMReX delivers compute performance on a variety of platforms and across a wide range of applications by default. While fine tuning hybrid MPI+OpeMP behavior is possible, it is not necessary to achieve performance.

**Comment by etpalmer63** [comment]
Kept the answered questions, going to move the unanswered to the Discussions page: https://github.com/AMReX-Codes/amrex/discussions

Had a bit of an adventure with git to get this cleaned up. I hope it is ok now.

File: Docs/sphinx_documentation/source/Faq.rst [hunk 2]
```diff
@@ -125,13 +125,35 @@ Source Documentation.) and
...
```
Comment (WeiqunZhang) [inline Docs/sphinx_documentation/source/Faq.rst]:
```suggestion
**Q.** How does coarse-grained OpenMP parallelism work in AMReX? How is it
different from the fine-grained approach?

**A.** Our OpenMP strategy is explained in this paper, https://arxiv.org/abs/1604.03570.
```

**Review by WeiqunZhang — APPROVED** [review]

**Pull request merged**